docs/release-notes/8.11.asciidoc

```diff
@@ -0,0 +1,62 @@
+[[release-notes-header-8.11.0]]
+== 8.11
+
+[discrete]
+[[release-notes-8.11.0]]
+=== 8.11.0
+
+[discrete]
+[[known-issue-8.11.0]]
+==== Known issues
+* For MITRE ATT&CK® technique cells:
+** Duplicate rules are displayed ({issue}167929[#167929]).
+** The incorrect rule count is shown ({issue}167930[#167930]).
```

Comment:
The incorrect rule count is technically in the tactic cell, not the technique cell. All these MITRE issues can probably be bullet points on the same level.